zhaquirks/xiaomi/aqara/vibration_aq1.py

```diff
@@ -70,15 +70,13 @@ class VibrationBasicCluster(BasicCluster):
         """Vibration cluster."""
 
         cluster_id = BasicCluster.cluster_id
-        attributes = Basic.attributes.copy()
-        attributes.update({0xFF0D: ("sensitivity", types.uint8_t)})
+        manufacturer_attributes = {0xFF0D: ("sensitivity", types.uint8_t)}
```

Comment:
this guy needs a different mfg code. I think it is 4447 but let me double check

Reply:
Yeah it is 4447

Reply:
Is it the same for all xiaomi or just for vibration sensor?

Reply:
In other words should it be overriden just for vibration sensor or all xiaomi? I think just for vibration for starters

Reply:
> Is it the same for all xiaomi or just for vibration sensor?

Just this one. It reports something different in the node descriptor

Reply:
Added override in https://github.com/zigpy/zha-device-handlers/pull/400/commits/b6373fa7a1a551bf412606404200cb873e1edc10